src/site/topic14.rst

```diff
@@ -22,6 +22,68 @@ Terminology
     * An ``Object`` is also a superclass of ``Bag``, ``SortedBag``, and ``IndexedBag``
 
 
+Polymorphism
+============
+
+* Polymorphism is the idea that behavior can change depending on the type of the object
+* We've already made use of this idea without having gone into any details
+    ``Stack<Integer> myStack = new ArrayStack<>();``
+    ``Stack<Integer> myStack = new LinkedStack<>();``
+
+* Our reference variables here is for something of type ``Stack``
+* Yet, it can reference an object of type ``ArrayStack`` or ``LinkedStack``
+* This is because, ``ArrayStack`` *is a* ``Stack`` and ``LinkedStack`` *is a* ``Stack``
+
+* You **cannot** however do this
+    ``ArrayStack<Integer> myStack = new LinkedStack<>();``
+
+* This is because a ``LinkedStack`` is **not** an ``ArrayStack``
```

Comment:
Another warning here about the _is a_ metaphor would be nice to see. _Is a_ is fine so long as you're refering to the types, not to the things they represent.